notebooks/community/generative_ai/text_embedding_api_cloud_next_new_models.ipynb

```diff
@@ -215,7 +215,7 @@
       },
```

Comment:
<p>Recommend to use pip3 for package installation:</p><p>! pip3 install --upgrade --quiet google-cloud-aiplatform</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2233/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>